wf-core-tck-runner/pom.xml

```diff
@@ -97,8 +102,8 @@
     <wildfly.plugin.version>3.0.0.Final</wildfly.plugin.version>
     <!-- WildFly JSON-P/JSON-B api jars -->
     <wildfly.home>${env.JBOSS_HOME}</wildfly.home>
-    <wildfly.json-api.jar>${wildfly.home}/modules/system/layers/base/jakarta/json/api/main/jakarta.json-api-2.1.1.jar</wildfly.json-api.jar>
-    <wildfly.json-bind-api.jar>${wildfly.home}/modules/system/layers/base/jakarta/json/bind/api/main/jakarta.json.bind-api-3.0.0.jar</wildfly.json-bind-api.jar>
+    <wildfly.json-api.jar>${wildfly.home}/${JSONP_API}</wildfly.json-api.jar>
+    <wildfly.json-bind-api.jar>${wildfly.home}/${JSONB_API}</wildfly.json-bind-api.jar>
```

Comment:
Don't these need to be prefixed with `env`? Like `${env.JSONP_API}`.

Reply:
I'll update